examples/alternate-descriptors/README_es.md

```diff
@@ -0,0 +1,162 @@
+indice-grupo=Técnicas de prueba
+tipo=página
+estado=publicado
+título=Descriptores alternos
```

Comment:
This and the above three lines are used for configuration so the properties should remain in English and you can translate the values of `index-group` and `title`. 
I propose to change the first four line to the following code, I tested successfully on my local deployment.

```
index-group=Testing Techniques
type=page
status=published
title=Descriptores alternos
```